.docker/entrypoint/Gemfile

```diff
@@ -7,16 +7,16 @@ group :jekyll_plugins do
   gem 'jekyll-assets'
   gem 'jekyll-github-metadata'
   gem 'jekyll-material-icon-tag'
-  gem 'jekyll-remote-theme'
   gem 'jemoji'
   gem 'kramdown-plantuml'
   gem 'rouge'
 end
 
+gem 'concurrent-ruby'
 gem 'docopt'
 # Neccessary to prevent Jekyll errors. See https://github.com/github/personal-website/issues/166
 gem 'faraday', '~> 1.0.1'
-gem 'html-proofer'
+gem 'html-proofer', git: 'https://github.com/NullableInt/html-proofer', branch: 'feature/cache_local_links'
```

Comment:
Codacy found an issue: [Line is too long. [107/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)